doc-app/tests/integration/components/ember-input-validation/tpk-form-test.ts

```diff
@@ -113,4 +115,88 @@ module('Integration | Component | tpk-form', function (hooks) {
 
     assert.verifySteps(['onSubmit']);
   });
+
+  test<ComponentTestContext>('Should have asterix in label on initiate and when adding element', async function (assert) {
```

Comment:
on initiate ?

Reply:
Should display an asterisk in the label upon initialization of the form and when adding an element

Correct pour toi ?